src/flashcard-form.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+export default class FlashcardForm extends React.Component {
+
+  render() {
+    return (
+      <div className="container-fluid w-50 border border-dark p-4 rounded">
```

Comment:
Dark borders are _super_ risky. I don't really recommend using them unless you are diligent about applying them everywhere. And even then, it's probably not going to look that great.